port/port_posix_sse.cc

```diff
@@ -50,11 +50,11 @@ static inline uint64_t LE_LOAD64(const uint8_t *p) {
 
 static inline bool HaveSSE42() {
 #if defined(_MSC_VER)
-  int cpu_info[4];
+  int cpu_info[4] = {};
   __cpuid(cpu_info, 1);
   return (cpu_info[2] & (1 << 20)) != 0;
 #elif defined(__GNUC__)
-  unsigned int eax, ebx, ecx, edx;
+  unsigned int eax, ebx, ecx=0, edx;
```

Comment:
I think this isn't the best way to silence this warning. :P  The warning was because the compiler is smart enough to see that __cpuid can fail. it'll still work but it's ugly, better to handle the failure like Bitcoin core does?